### PR TITLE
Buggy cities

### PR DESCRIPTION
**Pull request by berkserbet**
### Why:
* Fixed issue with some cities not showing any weather sensitive heating and cooling.

### What:
* Removed zip 833 from acceptable base cities. 833 contained negated heating and cooling sensitivities that were corrected to 0.
* Additional bug fix was made to calculation of projection vector using weather during projection.

### Checklist
- [x] Finished My Changes
- [ ] Automated Unit Tests

### CODE REVIEW

File: pipelines/ceus/tasks/project_loadshapes.py [hunk 1]
```diff
@@ -141,6 +141,8 @@ def get_multiplier(self, weather):
         for time_temp in weather:
             if self.theat < time_temp < self.tcool:
                 multiplier_array.append(0)
+            elif self.tcool < time_temp:
+                multiplier_array.append(time_temp-self.tcool)
```
Comment (jongoncalves) [inline pipelines/ceus/tasks/project_loadshapes.py]: put some space around the `-`

File: pipelines/rbsa/tasks/project_loadshapes.py [hunk 2]
```diff
@@ -135,6 +135,8 @@ def get_multiplier(self, weather):
         for time_temp in weather:
             if self.theat < time_temp < self.tcool:
                 multiplier_array.append(0)
+            elif self.tcool < time_temp:
+                multiplier_array.append(time_temp-self.tcool)
```
Comment (jongoncalves) [inline pipelines/rbsa/tasks/project_loadshapes.py]: same as above

File: pipelines/rbsa/tasks/zipcode_correlation.py [hunk 3]
```diff
@@ -49,12 +49,13 @@ def _task(self):
 
         for base in self.full_zipcodes:
             for target in self.projection_locations:
-                if target == 'CorpusChristi,TX':
-                    continue
+                # this section will need to be fixed so it reads loadshapes file and skips skewed base zipcodes.
                 if base == 97239:
                     continue
                 if base == 97008:
                     continue
```
Comment (jongoncalves) [inline pipelines/rbsa/tasks/zipcode_correlation.py]: You can join these conditionals:
```
if base in [97239, 97008]:
    ...
```

File: pipelines/ceus/tasks/project_loadshapes.py (hunk 1 above)
Comment (berkserbet) [reply pipelines/ceus/tasks/project_loadshapes.py]: fixed

File: pipelines/rbsa/tasks/project_loadshapes.py (hunk 2 above)
Comment (berkserbet) [reply pipelines/rbsa/tasks/project_loadshapes.py]: fixed

File: pipelines/rbsa/tasks/zipcode_correlation.py (hunk 3 above)
Comment (berkserbet) [reply pipelines/rbsa/tasks/zipcode_correlation.py]: fixed

**Pull request merged**